src/data/sym/basic.lean

```diff
@@ -150,4 +153,97 @@ instance inhabited_sym' [inhabited α] (n : ℕ) : inhabited (sym' α n) :=
 
 end inhabited
 
+instance has_zero : has_zero (sym α 0) := ⟨⟨0, rfl⟩⟩
+instance has_emptyc : has_emptyc (sym α 0) := ⟨0⟩
+
+instance subsingleton {n : ℕ} [subsingleton α] : subsingleton (sym α n) :=
```

Comment:
I would still make all the arguments explicit. Usually, you don't need to mention this declaration, because TC search takes care of everything for you. But if you do need to use it by hand, you probably want to provide the arguments explicitly as well.
```suggestion
instance subsingleton (n : ℕ) [subsingleton α] : subsingleton (sym α n) :=
```